utils.js

```diff
@@ -513,6 +513,11 @@ export const parsePkgLock = async (pkgLockFile, options = {}) => {
         "bom-ref": purlString
       };
     }
+    const packageLicense = node.package.license;
+    if (packageLicense) {
+      // License will be overridden if FETCH_LICENSE is enabled
+      pkg.license = packageLicense;
```

Comment:
Hmm, can we make fetch_license skip packages that already have the license?

Reply:
Good thought. I originally left the override in because I felt like the environment variable should override the input data, but thinking it over again, I don't like that data flow.  If it is possible for the input data to specify different or custom licenses that are not on the public registry (I'm not sure if this is possible), then we would be generating an incorrect BOM based on the input.

Reply:
Yes, it is tricky since either or both places could have the wrong license information. Let's keep it as-is for now!

Reply:
Looks like I just missed your comment before pushing. Please let me know if you'd like me to revert the latest change.

Reply:
I dont know what is the correct approach tbh. Also, we seem to be overwriting for other languages too so I am fine either ways.

Reply:
Then let's stick with consistency and keep the override behavior.